test/integration/targets/jboss/tasks/main.yml

```
@@ -0,0 +1,12 @@
+- name: Tests for jboss modules
+  block:
+
+  - import_tasks: jboss.yml
+    when: ansible_distribution == 'CentOS' and ansible_distribution_major_version >= '7'
```

Use the `version` test:
```suggestion
    when:
      - ansible_distribution == 'CentOS'
      - ansible_distribution_major_version is version('7', '>=')
```

---

I didn't know about this way, thanks for the hint, added to my check list